GenderGapsInCollegeDegrees.md

```diff
@@ -0,0 +1,160 @@
+
+## Clean up the chart
+1. Let's explore the gender gap in STEM fields, such as science, technology, engineering, and mathematics.
```

Comment:
It would be good to describe the dataset here, link to it, and include `pandas.head()` so people can see the first few rows.